docassemble/ALWeaver/interview_generator.py

```diff
@@ -24,21 +24,20 @@
 import docassemble.base.pdftk
 import datetime
 import zipfile
-import json
-from typing import Any, Dict, List, Optional, Set, Tuple, Union
+from typing import Any, Dict, List, Optional, Set, Tuple, Union, Iterable
 from .generator_constants import generator_constants
 from .custom_values import custom_values
 from .validate_template_files import matching_reserved_names
-import ruamel.yaml as yaml
 import mako.template
 import mako.runtime
 from pdfminer.pdfparser import PDFSyntaxError
 from pdfminer.psparser import PSEOF
+from pikepdf import Pdf
 from PyPDF2.utils import PdfReadError
 from zipfile import BadZipFile
 import ast
 from enum import Enum
-import itertools
+from itertools import zip_longest, chain
```

Comment:
This is fine, we don't have a style guide on point. I know Google specifically suggests using the fully qualified import to make it more clear where functions come from later in the code, but we certainly don't do it consistently anywhere.

Did you definitely catch all of the references to this though?

Reply:
Eh, readability is a balance, I prefer things to everything being on one line.

And pretty sure I did. Search doesn't show anything else, and mypy should catch simple import errors like that.

You might be thinking of `more_itertools`, which is a different package.